src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/FilterParser.peg

```diff
@@ -1,6 +1,6 @@
 ﻿@namespace Microsoft.Diagnostics.EventFlow
 @classname FilterParser
-@accessibility internal
```

Comment:
Why expose this as public now?

Reply:
Good catch, I think it was a leftover from some earlier experiments, but I tried again and as far as I can tell, this change is not necessary. I will revert it after pushing this PR.